src/plone/recipe/zope2instance/ctl.py

```diff
@@ -874,6 +879,43 @@ def help_adduser(self):
         print("adduser <name> <password> -- add a Zope management user")
 
 
+def serve_paste(app, global_conf, **kw):
+    sock = None
+    if 'prebound' in global_conf:
+        _sock = socket.fromfd(
+            int(global_conf['prebound']), socket.AF_INET, socket.SOCK_STREAM)
+        if six.PY2:
+            sock = socket.socket()
+            sock._sock = _sock
+        else:
+            sock = _sock
+        kw.update(sockets=[sock])
+    try:
+        serve(app, **kw)
```

Comment:
The `serve` function below does not except keyword arguments.

Reply:
Sorry, got confused, this one calls `waitress.serve`. Maybe it should not imported via a from-import to make clear which `serve` function is called.